modules/workflow-service-api/src/main/java/org/opencastproject/workflow/api/WorkflowInstanceImpl.java

```diff
@@ -202,8 +210,15 @@ public User getCreator() {
    * @see org.opencastproject.workflow.api.WorkflowInstance#getOrganization()
    */
   @Override
-  public Organization getOrganization() {
-    return organization;
+  public String getOrganization() {
```

Comment:
getOrganizationId() would be more consistent with other similar situations in the codecase